components/impls/core/src/test/javascript/cde/components/Map/amd/ControlPanel-spec.js

```diff
@@ -1,8 +1,21 @@
+/*!
+ * Copyright 2002 - 2018 Webdetails, a Hitachi Vantara company. All rights reserved.
+ *
+ * This software was developed by Webdetails and is provided under the terms
+ * of the Mozilla Public License, Version 2.0, or any later version. You may not use
+ * this file except in compliance with the license. If you need a copy of the license,
+ * please go to http://mozilla.org/MPL/2.0/. The Initial Developer is Webdetails.
+ *
+ * Software distributed under the Mozilla Public License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. Please refer to
+ * the license for the specific language governing your rights and limitations.
+ */
+
 define([
   'cdf/lib/jquery',
   'cdf/Dashboard.Clean',
-  'cde/components/Map/ControlPanel/ControlPanel',
-  'cde/components/Map/model/MapModel'
+  'cde/components/Map/amd/ControlPanel/ControlPanel',
```

Comment:
Are these AMD ids public or of internal components?

Reply:
Internal for tests.